decisions/wc_dark_cult_decisions.txt

```diff
@@ -637,11 +637,31 @@ decisions = {
 			prisoner = no
 			is_inaccessible_trigger = no
 			independent = yes
-			
+
 			government = black_pact_government
-			
+
 			religion_authority = 0.75
 			num_of_holy_sites = 4
+			trigger_if = {
+				limit = { cthun_offmap_decisions_enabled_trigger = yes }
+				offmap_cthun = { has_policy = old_god_working }
+				b_ahnqiraj = { holder_scope = { character = ROOT } }
+			}
+			trigger_else_if = {
+				limit = { nzoth_offmap_decisions_enabled_trigger = yes }
+				offmap_nzoth = { has_policy = old_god_working }
+				b_sagehold = { holder_scope = { character = ROOT } }
+			}
+			trigger_else = {
+				limit = { yogg_saron_offmap_decisions_enabled_trigger = yes }
+				offmap_yoggsaron = { has_policy = old_god_working }
+				b_ulduar = { holder_scope = { character = ROOT } }
```

Comment:
Should check `has_wonder_upgrade` now. **DO NOT FORGET TO ADD A SCRIPTED TRIGGER**! Like `has_cthun_prison_trigger`.